cmd/install.go

```
@@ -52,6 +52,7 @@ var installCmd = &cobra.Command{
 		cfg.Target = args[0]
 
 		cfg.DigestSetup()
+		cmd.SilenceUsage = true
```

Isn't better to use silence errors?

---

I am not sure which is the difference. After DigestSetup() if there was something wrong with the flags (aka use of incompatible flags) the error should be raised there. Even for now we are no doing any consistency validation on flags. So my understanding is that from that stage flags are already validated hence the usage message on errors has no sense.